scripts/generate.py

```diff
@@ -0,0 +1,16 @@
+#this script is to generate simple RPM spec files
+#spec => rpm files: run rpmbuild -bb *.spec
+#install rpm: yum localinstall /root/rpmbuild/RPMS/x86_64/*.rpm
+
+for x in range(100):
+    with open("{}pkg.spec".format(x), "w") as f:
+        f.write("Summary: Summary here\n")
+        f.write("Name: package_name"+str(x)+"\n")
+        f.write("Version: 0\n")
+        f.write("Release: 0\n")
+        f.write("License: Public\n")
+        f.write("Group: Applications/System\n")
+        f.write("Requires: bash\n")
+        f.write("%description\n")
+        f.write("Package description here\n")
+        f.write("%files")
```

Comment:
Is it OK to remove the dependency on Python, especially if this is supposed to run in the node itself? 

(Note: Applying the suggestion means we should rename the file to `.sh` -- it is not required to execute the script, but it will definitely confuse people if you leave `*.py`).

```suggestion
#/usr/bin/env bash

# This script is to generate simple RPM spec files:
# spec => rpm files: run rpmbuild -bb *.spec
# install rpm: yum localinstall /root/rpmbuild/RPMS/x86_64/*.rpm

for n in {1..100}; do
    cat >pkg-$(printf "%03d" ${n}).spec <<EOF
Name: package_name_${n}
Summary: Package number ${n}
Version: 0
Release: 0
License: Public
Group: Applications/System
Requires: bash
%description
Package number ${n}
%files
EOF
done
```